### PR TITLE
Add tanzcafe (static) for chris, refactor comment, remove unecessary space

### PR DESCRIPTION
**Pull request by paulpaul168**

### CODE REVIEW

File: chris.py [hunk 1]
```diff
@@ -11,6 +11,34 @@
...
+        Weekday.SUN,
+    ]:
+        events += weekly_event(
+            weekday,
+            DanceEvent(
+                starts_at=dateparser.parse("17:00"),
+                ends_at=dateparser.parse("18:00"),
+                name="Tanzcafe",
+                price_euro_cent=500,
+                description="5-Uhr-Tee in Wien",
```
Comment (flofriday) [inline chris.py]: **thought:** The description could be longer as the website too has more information.

File: dorner.py [hunk 2]
```diff
@@ -12,7 +12,7 @@
 # notice that the website changes.
 def create_perfections() -> list[DanceEvent]:
     events = []
-    # Every saturday and sunday evening
+    # Every friday evening
```
Comment (flofriday) [inline dorner.py:15]: **praise:** nice catch

File: dorner.py [hunk 3]
```diff
@@ -21,7 +21,7 @@ def create_perfections() -> list[DanceEvent]:
                 ends_at=dateparser.parse("22:15"),
                 name="Perfektion",
                 price_euro_cent=700,
-                description="Freitagsperfektion I TanzZeit ",
+                description="Freitagsperfektion I TanzZeit",
```
Comment (flofriday) [inline dorner.py]: **chore:** I think we can remove the I in the description, even though it is on the website but I just think it kinda is misleading.

**Review by flofriday — APPROVED** [review]

File: chris.py (hunk 1 above)
Comment (paulpaul168) [reply chris.py]: Done

File: dorner.py (hunk 3 above)
Comment (paulpaul168) [reply dorner.py]: I guess you might be right

**Pull request merged**